lib/units/websocket/index.js

```diff
@@ -305,6 +305,28 @@ module.exports = function(options) {
 
     new Promise(function(resolve) {
       socket.on('disconnect', resolve)
+        // Global messages for all clients using socket.io
+        //
+        // Device note
+        .on('device.note', function(data) {
+          return dbapi.setDeviceNote(data.serial, data.note)
+            .then(function() {
+              return dbapi.loadDevice(data.serial)
+            })
+            .then(function(device) {
+              if(device) {
+                io.emit('device.change', {
```

Comment:
This doesn't actually necessarily achieve what you want, since it will only inform clients being served by this instance of the websocket unit. There could be more than one `stf websocket` running. The real way to do it would be to make it go all the way to the processor, and then send it back... however, the processor currently does not process any messages from the application side, so it would require extra code there as well. I actually need it myself for another feature, so I might be adding that part soon. I think what you have here is a reasonable workaround for now.